src/main/java/io/reactiverse/pgclient/PgConnectOptions.java

```diff
@@ -340,7 +339,7 @@ public PgConnectOptions setEnabledSecureTransportProtocols(Set<String> enabledSe
   /**
    * Configure the connection with environment variables.
```

Comment:
Actually it configures from non-env variable as well when they are not set and the current doc does not reflect that

Reply:
https://github.com/reactiverse/reactive-pg-client/pull/87/commits/72303c6ec0eea12ee68b2fe1b1ac1da2c0e6fad6 is this appropriate?

Reply:
rather : initialize with the default options, if env variables are specified they take precedence over these options.

Reply:
Fixed as you said.